google/cloud/baremetalsolution/bare_metal_solution_connection.cc

```diff
@@ -86,80 +126,26 @@ BareMetalSolutionConnection::ListNetworks(
       StreamRange<google::cloud::baremetalsolution::v2::Network>>();
 }
 
-StatusOr<google::cloud::baremetalsolution::v2::Network>
-BareMetalSolutionConnection::GetNetwork(
-    google::cloud::baremetalsolution::v2::GetNetworkRequest const&) {
-  return Status(StatusCode::kUnimplemented, "not implemented");
-}
-
-StreamRange<google::cloud::baremetalsolution::v2::SnapshotSchedulePolicy>
-BareMetalSolutionConnection::ListSnapshotSchedulePolicies(
```

Comment:
nice. [breaking changes](https://github.com/googleapis/googleapis/commit/7b1657f015de5214f3aa235168eafd5534f10551). The PR says "the removed Snapshots methods were never officially released". I wonder how we were supposed to know that.